databuilder/models/user_elasticsearch_document.py

```diff
@@ -0,0 +1,62 @@
+import json
+from typing import List, Optional  # noqa: F401
+
+from databuilder.models.elasticsearch_document import ElasticsearchDocument
+
+
+class UserESDocument(ElasticsearchDocument):
+    """
+    Schema for the Search index document for user
+    """
+    def __init__(self,
+                 elasticsearch_index,  # type: str
+                 elasticsearch_type,   # type: str
+                 email,  # type: str
+                 first_name='',  # type: str
+                 last_name='',  # type: str
+                 name='',  # type: str
+                 github_username='',  # type: str
+                 team_name='',  # type: str
+                 employee_type='',  # type: str
+                 manager_email='',  # type: str
+                 slack_id='',  # type: str
+                 is_active=True,  # type: bool
+                 total_read=0,  # type: int
+                 total_own=0,  # type: int
+                 total_follow=0,  # type: int
+                 ):
+        # type: (...) -> None
+        self.elasticsearch_index = elasticsearch_index
+        self.elasticsearch_type = elasticsearch_type
+        self.email = email
+        self.first_name = first_name
+        self.last_name = last_name
+        self.name = name
+        self.github_username = github_username
+        self.team_name = team_name
+        self.employee_type = employee_type
+        self.manager_email = manager_email
+        self.slack_id = slack_id
+        self.is_active = is_active
+        self.total_read = total_read
+        self.total_own = total_own
+        self.total_follow = total_follow
```

Comment:
Should we add a docstring or inline comment to explain what each of these attributes represent? One example for confusion here is, does `total_follow` here mean number of tables this user is interested in or number of users they are following or is it both?